charts/agent-operator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: grafana-agent-operator
 description: A Helm chart for Grafana Agent Operator
 type: application
-version: 0.1.13
+version: 0.2.0
 appVersion: "0.24.2"
 home: https://grafana.com/docs/agent/latest/
 icon: https://raw.githubusercontent.com/grafana/agent/v0.24.2/docs/assets/logo_and_name.png
```

Comment:
as mentioned above, may as well bump agent version at the same time (but not necessary)

Reply:
commented above about the agent version bump